pkg/controller/wip/controller.go

```diff
@@ -382,16 +387,30 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 	glog.V(4).Infof("Creating client for Broker %v, URL: %v", broker.Name, broker.Spec.URL)
 	brokerClient := c.brokerClientCreateFunc(broker.Name, broker.Spec.URL, username, password)
 
-	// TODO: parameters
+	parameters, err := unmarshalParameters(instance.Spec.Parameters.Raw)
```

Comment:
hmmm, if we have no params I think unmarshalParameters() will still generate a map, but an empty one. Which I think will result in an empty Parameters property, not a nil one. Will this cause us to generate an empty json field instead of skipping the field all together in the json?